The explanation about the version number property in config of the plugin (#3009)
cypress-io/cypress-documentation: source/api/plugins/writing-a-plugin.md

* `configFile`: The absolute path to the config file. By default, this is `<projectRoot>/cypress.json`, but may be a custom path or `false` if using the {% url "`--config-file` flag" command-line#cypress-open-config-file-lt-config-file-gt %}.
* `projectRoot`: The absolute path to the root of the project (e.g. `/Users/me/dev/my-project`)

> * `version`: The version number of the installed Cypress. It can be used to handle breaking changes.

Reviewer: ```suggestion
* `version`: The version number of Cypress. This can be used to handle breaking changes.
```
